src/sentry/utils/outcomes.py

```diff
@@ -71,11 +71,41 @@ def track_outcome(
     assert isinstance(category, (type(None), DataCategory))
     assert isinstance(quantity, int)
 
+    outcomes_config = settings.KAFKA_TOPICS[settings.KAFKA_OUTCOMES]
+    billing_config = settings.KAFKA_TOPICS.get(settings.KAFKA_OUTCOMES_BILLING) or outcomes_config
```

Comment:
This moved to inside the function now. @untitaker mentioned that it may even be faster than accessing globals several times. The main motivation was readability and testability, though.